### PR TITLE
Cross-platform regression tests

### PR DESCRIPTION
**Pull request by lithp**

### CODE REVIEW

File: src/test/regress/sql/multi_utility_statements.sql [hunk 1]
```diff
@@ -101,7 +101,7 @@ COPY (
 -- check copying to file
 -- (quiet off to force number of copied records to be displayed)
 \set QUIET off
-COPY nation TO '/dev/null';
+\COPY nation TO :dev_null;
```
Comment (pykello) [inline src/test/regress/sql/multi_utility_statements.sql]: When I run the tests, a file named `:dev_null` is created in the current directory. Maybe this variable is not passed to all test scripts.

File: src/test/regress/output/multi_copy.source [hunk 2]
```diff
@@ -640,8 +640,8 @@ SELECT master_create_worker_shards('super_packed_numbers_hash', 4, 1);
  
 (1 row)
 
-COPY (SELECT 1, ROW(ROW(42, 42), ROW(42, 42))) TO '/tmp/copy_test_composite_of_composite';
-COPY super_packed_numbers_hash FROM '/tmp/copy_test_composite_of_composite';
+COPY (SELECT 1, ROW(ROW(42, 42), ROW(42, 42))) TO '@testtablespace@\copy_test_composite_of_composite';
```
Comment (pykello) [inline src/test/regress/output/multi_copy.source]: Why are we using `@testtablespace@` as the temp directory? Can't we use something like `mktemp` which seems to be cross-platform?

File: src/test/regress/expected/isolation_add_node_vs_reference_table_operations.out [hunk 3]
```diff
@@ -5,7 +5,7 @@ create_reference_table
 
                
 step s2-load-metadata-cache: 
-	COPY test_reference_table FROM PROGRAM 'echo "1\n2\n3\n4\n5"';
+	COPY test_reference_table FROM PROGRAM 'echo 1 && echo 2 && echo 3 && echo 4 && echo 5';
```
Comment (pykello) [inline src/test/regress/expected/isolation_add_node_vs_reference_table_operations.out]: These files also have similar echos which have been missed from the current PR:
- isolation_create_distributed_table
- isolation_master_append_table
- isolation_master_apply_delete

File: src/test/regress/sql/multi_utility_statements.sql (hunk 1 above)
Comment (lithp) [reply src/test/regress/sql/multi_utility_statements.sql]: Addressed by 9436138

File: src/test/regress/expected/isolation_add_node_vs_reference_table_operations.out (hunk 3 above)
Comment (pykello) [reply src/test/regress/expected/isolation_add_node_vs_reference_table_operations.out]: What do you think about changing the following files?

    isolation_create_distributed_table
    isolation_master_append_table
    isolation_master_apply_delete

Comment (pykello) [reply src/test/regress/expected/isolation_add_node_vs_reference_table_operations.out]: Ok, I did this. Please take a look.

**Comment by pykello** [comment]
@lithp why are the tests failing for this branch? Otherwise, looks good to me once the failures are resolved.

**Pull request merged**